tsdb/index/index.go

```diff
@@ -425,65 +452,124 @@ func (w *Writer) WriteLabelIndex(names []string, values []string) error {
 		offset: w.pos,
 	})
 
-	w.buf2.Reset()
-	w.buf2.PutBE32int(len(names))
-	w.buf2.PutBE32int(valt.Len())
+	startPos := w.pos
+	if err := w.write([]byte("alen")); err != nil {
```

Comment:
Can we have same comment for this as we have for rest of the places?

Reply:
Done